.env

```diff
@@ -6,7 +6,7 @@ DATABASE_PASSWORD=docker
 DATABASE_HOST=db
 DATABASE_PORT=5432
 DATABASE_DB=airbyte
-DATABASE_URL=jdbc:postgresql://${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DB}
+DATABASE_URL=jdbc:postgresql://db:5432/airbyte
```

Comment:
```suggestion
DATABASE_URL=jdbc:postgresql://airbyte-db:5432/airbyte
```

Reply:
The docker service is just called `db`, just like `DATABASE_HOST` above.

Reply:
oh, i thought you need to use the docker container name